applications/serial_lte_modem/src/slm_at_host.c

```diff
@@ -258,9 +296,8 @@ bool exit_datamode(int result)
 		datamode_handler = NULL;
 
 		/* reset UART for command mode */
-		uart_rx_disable(uart_dev);
-		k_sleep(K_MSEC(10));
-		(void)uart_receive();
+		uart_receive_disable(false);
+		(void)uart_receive_enable();
```

Comment:
Why the return value is not checked here?

Reply:
Return value check added.